includes/polling/os/cnos.inc.php

```diff
@@ -0,0 +1,18 @@
+<?php
+/*
+ * Lenovo CNOS information module
+ *
+ * Copyright (c) 2020 Florian Zillner <zillo83@gmail.com>
+ * Parts Copyright (c) 2015 Søren Friis Rosiak <sorenrosiak@gmail.com>
+ * This program is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the
+ * Free Software Foundation, either version 3 of the License, or (at your
+ * option) any later version.  Please see LICENSE.txt at the top level of
+ * the source code distribution for details.
+ */
+$sysdescr_value = $device['sysDescr'];
```

Comment:
these should be converted to the new OS format, see for example https://github.com/librenms/librenms/blob/master/LibreNMS/OS/Endrun.php